app/controllers/registrations_controller.rb

```diff
@@ -0,0 +1,7 @@
+class RegistrationsController < Devise::RegistrationsController
+  def new
+    @arrow_path = root_path
+    @path_description = 'REGISTER'
```

Comment:
- [ ] When I click on sign up, I get an error. This is for the deployed version of your app. Check the console log. Error status 500
![Screen Shot 2023-10-28 at 3 00 16 PM](https://github.com/mohashyne/Budget_App/assets/62849872/5e2cf317-e0a9-44c2-be42-12d856bcd42e)

Reply:
fixed